pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -23,20 +23,20 @@ let
     ];
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "liquibase";
   version = "4.29.1";
 
   src = fetchurl {
-    url = "https://github.com/liquibase/liquibase/releases/download/v${version}/${pname}-${version}.tar.gz";
+    url = "https://github.com/liquibase/liquibase/releases/download/v${finalAttrs.version}/liquibase-${finalAttrs.version}.tar.gz";
     hash = "sha256-MFJP8cG+GqxGt3S8x+LVSI6yF8F06f+C8LrCRP65sRc=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ jre ];
 
   unpackPhase = ''
-    tar xfz ${src}
+    tar xfz ${finalAttrs.src}
   '';
```

Comment:
This shouldn't be needed.

Reply:
You're right, the package was successfully installed without the explicit `unpackPhase`. I removed it in a force push.